lisa/tools/ssh.py

```diff
@@ -47,7 +49,7 @@ def enable_public_key(self, public_key: str) -> None:
     def add_known_host(self, ip: str) -> None:
         self.node.execute(f"ssh-keyscan -H {ip} >> ~/.ssh/known_hosts", shell=True)
 
-    def get_sshd_config_path(self) -> str:
+    def get_default_sshd_config_path(self) -> str:
```

Comment:
Why changes the name? If it needs to support different paths, we can add a method to set path into the ssh class. The ssh class is hold by the node, so we can save value in it.